packages/atomic/stencil.config.ts

```diff
@@ -247,3 +249,16 @@ export const config: Config = {
     enableImportInjection: true,
   },
 };
+
+function externalizeDependenciesPlugin() {
+  return {
+    name: 'externalize-dependencies',
+    resolveId(source: string) {
+      // TODO: add bueno once it's published on the cdn and npm
```

Comment:
Since atomic uses bueno directly, we can't externalize unless we publish it to the CDN first.